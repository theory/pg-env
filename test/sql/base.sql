CREATE EXTENSION envvar;
SELECT COALESCE(length(get_env('HOME')), 0) >= 0;
SELECT get_env('no such envvar');
SELECT get_env(NULL);
