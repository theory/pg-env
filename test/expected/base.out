CREATE EXTENSION envvar;
SELECT COALESCE(length(get_env('HOME')), 0) >= 0;
 ?column? 
----------
 t
(1 row)

SELECT get_env('no such envvar');
 get_env 
---------
 
(1 row)

SELECT get_env(NULL);
 get_env 
---------
 
(1 row)

